app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -86,7 +86,7 @@ def inspect_work
       # and to provide depositor (used on app/views/hyrax/base/_form_share.html.erb)
       def build_change_set(resource)
         change_set_class.new(resource,
-                             depositor: current_user.user_key,
+                             depositor: [current_user.user_key],
```

Comment:
Should this be the responsibility of the change_set_class? From the parameter names, I would assume that the `depositor` would be singular.

Reply:
I feel that this is really a bug in valkyrie. Valkyrie supports sending scalar messages to fields that are defined as sets. In Valkyrie everything is a set.

Reply:
https://github.com/samvera-labs/valkyrie/issues/333